.github/workflows/diagnostics-reusable.yml

```
@@ -247,6 +248,71 @@ jobs:
+            echo "manifest=${manifest}" >> "${GITHUB_OUTPUT}"
+          fi
+
+      - name: Write performance regression report
+        if: ${{ always() }}
+        shell: bash
+        env:
+          RUN_ROOT: ${{ steps.meta.outputs.run_root }}
+          CURRENT_MANIFEST: ${{ steps.meta.outputs.run_root }}/run-manifest.json
+          BASELINE_MANIFEST: ${{ steps.performance_baseline.outputs.manifest }}
+          BASELINE_ARTIFACT: ${{ steps.performance_baseline.outputs.artifact_name }}
+          SKIP_REASON: ${{ inputs.profile == 'extended' && 'extended profile remains report-only for performance budgets until #718 resolves all-failed bank stress semantics.' || '' }}
```

actionlint (1.7.12)

[error] got unexpected EOF while lexing end of string literal, expecting '''

(expression)